### PR TITLE
Completed prep-css-flexbox. Vickram Kahlon

### PR DESCRIPTION
**Pull request by vkahlon**
flexbox 1
flexbox 2
flexbox 3
flexbox 4

Note: I use a 1440p 21:9 ultrawide monitor which is why the search engine, tabs, and the overall bar are long.

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-basis: calc(100% / 3);
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html:14]: Careful, `flex-basis` is meant to be used on items within a "flex container" (in this case the `div#card-deck`). Try creating a separate rule for `.card-container` and apply the `flex-basis` there.

File: prep-css-flexbox/display-flex.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - display: flex</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-direction;
```
Comment (thebearingedge) [inline prep-css-flexbox/display-flex.html:14]: You can remove this `flex-direction` as the default for `display: flex;` is `row` without having to include the property.

File: prep-css-flexbox/flex-wrap.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - flex-wrap</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+        flex-basis: calc(100% / 6);
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html:14]: Same here regarding the `flex-basis`

File: prep-css-flexbox/justify-content.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - justify-content</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-basis: 25%;
```
Comment (thebearingedge) [inline prep-css-flexbox/justify-content.html:14]: Here as well for the `flex-basis`

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-flexbox/justify-content.html (hunk 4 above)
Comment (vkahlon) [reply prep-css-flexbox/justify-content.html]: reuploaded and made the necessary changes. thanks for the input!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**